src/app/components/files/uploader/component.ts

```diff
@@ -59,10 +61,11 @@ export class FileUploaderComponent {
   }
 
   private success(x) {
-    console.log(x)
     this.cancel()
     this.progress = 0
     this.form.enable()
+
+    this.fileViewer.addImage(x.data)
```

Comment:
```suggestion
    this.fileViewer.updateImages()
```

*see comments below*